src/components/carousel/index.jsx

```diff
@@ -129,6 +204,13 @@ const Carousel = ({
 			</div>
 
 			<div className="c-carousel__actions">
+				{/* only show button at all if enabled on the document */}
+				{fullScreenEnabledAllowed && !isFullScreen ? resolvedFullScreenShowButton : null}
```

Comment:
The full screen button should be placed at the top of the carousel - adding them into this actions container isn't the correct choice and will cause issues with styling and tab order.
We will want to have a container that sits at the top of the carousel to house this feature, auto play and the counter

Reply:
ok thanks, updated @matthewroach